vies/fields.py

```diff
@@ -31,9 +31,10 @@ def compress(self, data_list):
             value = ''.join(data_list)
             try:
                 vatin = VATIN(*data_list)
+                is_valid = vatin.is_valid()
             except ValueError as e:
```

Comment:
Agreed. A `ViesCheckUnavailable` exception or something alike would be better. It also avoids catching other errors by accident. Shall I change it into that?

Another point (for a separate PR): the current code doesn't check what the contents of the `WebFault` is.